.prettierignore

```diff
@@ -2,6 +2,7 @@
 bundles
 dist
 fixtures
+output
```

Comment:
Out of curiosity, what generates output?

Reply:
Narrowed to `ses-integration-test`.